editor/src/clj/editor/gui.clj

```diff
@@ -2652,8 +2709,8 @@
                                                                                   (into {})))))]]
                         (g/make-nodes graph-id [layout [LayoutNode (dissoc layout-desc :nodes)]]
                                       (attach-layout self layouts-node layout)
-                                      (g/set-property layout :nodes (:nodes layout-desc)))))))
-     custom-data)))
+                                      (g/set-property layout :nodes (:nodes layout-desc))))))
+      custom-data)))
```

Comment:
Indent / paren level adjusted due to getting rid of superfluous wrapping `conj` expression.